### PR TITLE
Add Rector.

### PR DESCRIPTION
**Pull request by Aerendir**
Waiting for https://github.com/rectorphp/rector/issues/2724 to be completed.

### CODE REVIEW

**Comment by Aerendir** [comment]
WIP

File: data/repositories.xml [hunk 1]
```diff
@@ -111,4 +111,7 @@
     <phar alias="composer-normalize" composer="ergebnis/composer-normalize">
         <repository type="github" url="https://api.github.com/repos/ergebnis/composer-normalize/releases" />
     </phar>
+    <phar alias="rector" composer="rector/rector-prefixed">
+        <repository type="github" url="https://api.github.com/repos/erectorphp/rector-prefixed/releases" />
```
Comment (theseer) [inline data/repositories.xml]: erectorphp? ;-)

Comment (Aerendir) [reply data/repositories.xml]: A typo! 😅Theank you... I will fix it!

Comment (theseer) [reply data/repositories.xml]: :)

**Comment by theseer** [comment]
Since the referenced ticket at the rector project has been closed without providing a signed release, this PR is kind of stuck.

Feel free to open a new PR if the prerequisite for an alias registration has been met.

**Comment by mzeis** [comment]
Hi @theseer, do I understand correctly that it would be an option - even if suboptimal - to add Rector to phar.io with unsigned phars? Rector provides a rector.phar files and tags releases in https://github.com/rectorphp/rector-prefixed.

**Comment by theseer** [comment]
No, we only add aliases to projects who sign their releases.

But you do not need an alias to install things with `phive` as long as the used specifier can be resolved, like `rectorphp/rector-prefixed` for instance. 

There are no phar files attached to the releases at `rectorphp/rector-prefixed` though, so this won't work.

**Comment by mzeis** [comment]
Thank you for the explanation!

**Pull request closed without merging**